doc/doxygen/MainPage.h

```diff
@@ -1,88 +1,200 @@
- * |--------------------------------|------------------------------------------------------|
- * | arrow keys                     | move, steer, accelerate, decelarate                  |
- * | mouse+rightclick               | rotate camera                                        |
- * | mousewheel+rightclick          | camera zoom                                          |
- * | C                              | change camera perspective                            |
- * | T                              | display vehicle stats                                |
- * | BACKSPACE                      | reset vehicle at current location                    |
- * | I                              | reset vehicle to spawnpoint                          |
- * | TAB                            | show minimap                                         |
- * | ESC                            | show menu                                            |
- * | A/Z                            | trucks: shift up/down                                |
- * | CTRL+HOME                      | aircrafts: start engine                              |
- * | Page Up/Down                   | aircrafts: increase/decrease throttle                |
- * | F1/F2                          | helicopters: lift up/down                            |
- * | Print Screen                   | create screenshot in $user/ folder                   |
- * 
- * 
- * ## Content/Mods
- * Rigs of Rods only comes with a very small selection of vehicles and terrains. For the best experience download some mods from the [Rigs of Rods Mod Repository](http://www.rigsofrods.org/repository/). The [Showroom Subforum](http://www.rigsofrods.org/forums/103-Showrooms-and-WIP) may contain additional content not found in the Mod Repository.  
- * If you want to get going quickly have a look at modpacks which can be found in the Mod Repository as well.
- * 
- * 
- * ## Configuration files
- * - Default location:  
- *  $user/config/* 
- * - They are created by RoRConfig on first use
- * 
- * 
- * ## Command-line options
- * 
- * * -map \<mapname\>
- * * loads map \<mapname\> on startup. Example:
- *  * *  RoR.exe -map aspen 
- * * note: do not add .terrn file format extension
- * * -truck \<truckfile\>
- * * loads a truck on startup. Example: 
- *  * *  RoR.exe -map oahu -truck semi.truck 
- *  * *  RoR.exe -map oahu -truck an-12.airplane 
- * * -enter
- * * enter selected truck by -truck option on startup
- * * -setup 
- * * displays OGRE3D settings dialog instead of loading settings from ogre.cfg 
- * * -help
- * * displays help for command line arguments
- * 
- *  
- * 
- * ## License of Rigs of Rods
- * 
  * > Copyright (c) 2005-2013 Pierre-Michel Ricordel  
- * > Copyright (c) 2007-2013 Thomas Fischer  
- * > Copyright (c) 2009-2015 Rigs of Rods Contributors
+ * > Copyright (c) 2007-2013 Thomas Fischer
+ * > Copyright (c) 2013-2022 Thomas Fischer
```

Comment:
Thomas is shown twice here. Should be Copyright (c) 2013-2022 Petr Ohlidal (& contributors).